docs/source/manual/jooq.rst

```diff
@@ -112,9 +112,85 @@ Below is an example of configuring the required binding uses a C3P0 pooling data
     }
   }
 
+Post commit actions
+~~~~~~~~~~~~~~~~~~~~
+
+Sometimes you need to perform an action only after the current transaction successfully commits. For example,
+updating or invalidating a cache.
+
+The JooqTransactionalHooks_ class provides the :code:`whenCommitted` method for registering callback actions that should be executed after a transaction is successfully committed. Any no-arguments lambda expression or :code:`Runnable` implementation can be passed as the action.
+
+.. code-block:: java
+
+  hooks.whenCommitted(() -> cache.invalidate())
+
+The action you register will be called immediately after the current transaction is successfully committed.
+
+If you call :code:`whenCommitted` while there isn't an active transaction, the action will be executed immediately.
+
+If the current transaction is rolled back instead of committed, then your action will be discarded and never called.
+
+Nested Transactions
+"""""""""""""""""""
+
+Nested transactions (savepoints) are handled correctly. Actions registered via :code:`whenCommitted` in a nested transaction will only be called after the outermost transaction is committed, and will not be called if a rollback to any outer savepoint occurs during the transaction.
```

Comment:
I think that `Nested transactions (savepoints) are handled correctly. ` can be removed.

Reply:
https://github.com/cerner/beadledom/pull/140/commits/9fb82820a6b7e1c326a58907464a9fdacce4eb1f